### DISCUSSION TITLE
FF Nightly 119 disabled TMP 1.0.0-pre.17

### DISCUSSION THREAD

**Discussion by heram01** [onemen/TabMixPlus#238] [Q&A]
Mac OS 13.3 Updating to FF Nightly to 119 disables TMP 1.0.0-pre.17.
All FF's possible (recommended) settings are set to "TRUE". 
The TMP perfectly works with FF 116.0a1 (2023-06-21) - either previous version of TMP or the latest one (3 days ago). 
But the moment I update the FF to 119 - it disables the TMP and no way (for me) get it enabled again.
The only way to restore both FF Nightly and the TMP - it's to downgrade the FF (and its profile) to previously working version by using the Time Machine.
May anybody confirm that it works for them in Mac OS and what I should do to make it work on my system?
Thank you.

**Comment by onemen** [comment]
Read this https://github.com/onemen/TabMixPlus/discussions/237

**Reply by heram01** [reply]
Thank you, I try to follow all these steps (while FF 116.03 works, it's 119 which disables the TMP).
But I do not understand this:
"Profile Folder is the path to create chrome folder and install [utils] folder in it".
First of all - what/where is that Profile Folder path in MacOS ? Which one? Second of all - what does it mean "install" [utils]? - is it just copy them? What where is the "chrome" folder? Does it have a name "chrome" - it doesn't exist on my Mac.
Please, may you just provide the paths or commands which would reveal those complete paths and directory (folder) name which should be created and whether the "util's" contect copied in it or the subdirectory "util" should copied in it.

It's really very hard every time restore older FF (116) and its profile while losing the current content of the day.

Thank you very much!

**Reply by onemen** [reply]
https://github.com/onemen/TabMixPlus/discussions/237#discussioncomment-6933398
